dll/win32/shlwapi/ordinal.c

```diff
@@ -4121,6 +4121,33 @@ BOOL WINAPI IsOS(DWORD feature)
     return FALSE;
 }
 
+#ifdef __REACTOS__
+/*************************************************************************
+ * @  [SHLWAPI.438]
+ */
+HRESULT WINAPI SHLoadRegUIStringA(HKEY hkey, LPCSTR value, LPSTR buf, DWORD size)
+{
+    WCHAR valueW[MAX_PATH], bufferW[MAX_PATH];
+    DWORD sz = ARRAY_SIZE(bufferW) * sizeof(CHAR);
```

Comment:
Use "dwSize", or just "size" instead. (sz means "string zero terminated", usually)

Reply:
`dwSize` is used.